aws-rds-dbcluster/docs/README.md

```diff
@@ -266,7 +266,7 @@ _Required_: No
 
 _Type_: String
 
-_Allowed Values_: <code>aurora</code> | <code>aurora-mysql</code> | <code>aurora-postgres</code>
+_Allowed Values_: <code>aurora</code> | <code>aurora-mysql</code> | <code>aurora-postgresql</code>
```

Comment:
From the documentation for create-db-cluster: 
```
Valid Values: https://docs.aws.amazon.com/cli/latest/reference/rds/create-db-cluster.html
    aurora (for MySQL 5.6-compatible Aurora)
    aurora-mysql (for MySQL 5.7-compatible and MySQL 8.0-compatible Aurora)
    aurora-postgresql
    mysql
    postgres
```

Does cfn only support aurora based engines?